OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -652,7 +652,7 @@ int IncreaseAltitude(int predClfDist, int diffClfMslHgt, int relTarHorDist, int
 					else
 					{
 						// Aim for the target
-						var vDist = new WVec(-relTarHgt, -relTarHorDist * (targetPassedBy ? -1 : 1), 0);
+						var vDist = new WVec(-relTarHgt, -relTarHorDist, 0);
```

Comment:
Wouldn't that mean that `targetPassedBy` is always `true` when we reach this code? In that case you need to keep the `* -1`, i.e. just remove the `-` infront of `relTarHorDist`.